src/main/java/com/google/sps/CapstoneAuth.java

```diff
@@ -0,0 +1,44 @@
+package com.google.sps;
+
+import java.io.IOException;
+
+import com.google.firebase.FirebaseApp;
+import com.google.firebase.FirebaseOptions;
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.auth.FirebaseToken;
+import com.google.firebase.auth.FirebaseAuthException;
+
+import com.google.auth.oauth2.GoogleCredentials;
+
+public final class CapstoneAuth {
+
+    private static CapstoneAuth currentInstance;
+
+    private CapstoneAuth() {
+        try {
+            FirebaseOptions options = new FirebaseOptions.Builder()
+                .setCredentials(GoogleCredentials.getApplicationDefault())
+                .setDatabaseUrl("https://chap-2020-capstone.firebaseio.com/")
```

Comment:
Could it make sense to set this URL as some variable so there aren't 'magic URL's"